test/foundry/unit/RewardEscrowV2/RewardEscrowV2.t.sol

```diff
@@ -263,24 +263,12 @@ contract RewardEscrowV2Tests is DefaultStakingV2Setup {
         rewardEscrowV2.createEscrowEntry(address(this), 0, 52 weeks, 90);
     }
 
-    function test_createEscrowEntry_Should_Not_Append_Entries_With_Short_Duration() public {
```

Comment:
**Thought:** If you make the suggested changes to the `EscrowMigrator`, a version of this test may still apply